README.md

```diff
@@ -25,7 +25,7 @@
 ## Compile and Run
 > See https://mpm-doc.cb-geo.com/ for more detailed instructions. 
 
-0. Run `mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release /path/to/CMakeLists.txt`.
+0. Run `mkdir build && cmake -B /path/to/build/folder -DCMAKE_BUILD_TYPE=Release /path/to/CMakeLists.txt`.
```

Comment:
Could you leave it as such. You create a build directory, you can `cd` to build to compile and build the files.

Reply:
OK! Understood!